data-migration/migrate-mysql-tidb-above-tb.md

```diff
@@ -0,0 +1,273 @@
+    backend = "local"
+    # For better import performance, it is recommended to use a directory different from `data-source-dir` and use flash storage and exclusive I/O for the directory.
+    sorted-kv-dir = "${sorted-kv-dir}"
+
+    [mydumper]
+    # The data source directory. The same directory where Dumpling exports data in Step 1.
+    data-source-dir = "${data-path}"
+
+    # Configures the wildcard rule. The default rule filters out all tables in these system databases: mysql, sys, INFORMATION_SCHEMA, PERFORMANCE_SCHEMA, METRICS_SCHEMA, INSPECTION_SCHEMA.
+    # If you do not configure this item, TiDB Lightning reports a `cannot find schema` exception when it imports system tables.
+    filter = ['*.*', '!mysql.*', '!sys.*', '!INFORMATION_SCHEMA.*', '!PERFORMANCE_SCHEMA.*', '!METRICS_SCHEMA.*', '!INSPECTION_SCHEMA.*']
+
+    [tidb]
+    # The target TiDB cluster information.
+    host = ${host}                # e.g.: 172.16.32.1
+    port = ${port}                # e.g.: 4000
+    user = "${user_name}"         # e.g.: "root"
+    password = "${password}"      # e.g.: "rootroot"
+    status-port = ${status-port}  # Obtains the table schema information from TiDB status port, e.g.: 10080
+    pd-addr = "${ip}:${port}"     # The cluster PD address, e.g.: 172.16.31.3:2379. TiDB Lightning obtains some information from PD. When backend = "local", you must specify status-port and pd-addr correctly. Otherwise, the import will be abnormal.
+    ```
+
+    For more information on TiDB Lightning configuration, refer to [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
+
+2. Start the import by running `tidb-lightning`. If you launch the program directly in the command line, the program might exit because of the `SIGHUP` signal. In this case, it is recommended to run the program using a `nohup` or `screen` tool. For example:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    nohup tiup tidb-lightning -config tidb-lightning.toml > nohup.out &
+    ```
+
+3. After TiDB Lightning completes the import, it exits automatically. If the import is successful, the last line of `tidb-lightning.log` prints `tidb lightning exit`.
+
+If the import fails, refer to [TiDB Lightning FAQ](/tidb-lightning/tidb-lightning-faq.md) for troubleshooting.
+
+## Step 3. Replicate incremental data to TiDB
+
+### Add the data source
+
+1. Create a `source1.yaml` file as follows:
+
+    {{< copyable "" >}}
+
+    ```yaml
+    # Configuration.
+    source-id: "mysql-01" # Must be unique.
+
+    # Configures whether DM-worker uses the global transaction identifier (GTID) to pull binlogs. To enable this mode, the upstream MySQL must also enable GTID. If the upstream MySQL has automatic source-replica switching, GTID mode is required.
```

Comment:
`If the upstream MySQL service is configured to switch master between different nodes automatically, GTID mode is required`